### PR TITLE
Update/r integration

### PR DESCRIPTION
**Pull request by jakubczakon**
* divided int quickstart and main example
* added video
* updated example scripts

### CODE REVIEW

File: docs/execution-environments/r-support.rst [hunk 1]
```diff
@@ -1,178 +1,278 @@
...
+    create_experiment(name='minimal example')
+
+This also creates a link to the experiment. Open the link in a new tab.
+The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
+
+.. note::
+
+    Neptune logs your hardware consumption and console outputs automatically.
+
+    Go to the ``Monitoring`` in your Neptune experiment to see it.
```
Comment (hubertjaworskineptune) [inline docs/execution-environments/r-support.rst:110]: If the user does not have psutils installed, monitoring tab will be empty, maybe properties tab?

Comment (hubertjaworskineptune) [reply docs/execution-environments/r-support.rst:110]: Or at least mention that user has to have it installed to see this

File: docs/execution-environments/r-support.rst [hunk 2]
```diff
@@ -1,178 +1,278 @@
...
 
-        init_neptune(project_name = 'my_workspace/my_project',
-                     api_token = Sys.getenv('NEPTUNE_API_TOKEN')
-                     )
+    If you are using R studio you can just follow along and you will be prompted by the console to install miniconda.
 
-**Specifying Python environment**
+Step 1: Install Neptune package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Install ``neptune`` package for R from CRAN.
```
Comment (hubertjaworskineptune) [inline docs/execution-environments/r-support.rst:67]: Make sure to note that this should be done within R interactive console, not bash - we had a user that missed that

**Review by hubertjaworskineptune — CHANGES_REQUESTED** [review]

File: docs/execution-environments/r-support.rst (hunk 1 above)
Comment (jakubczakon) [reply docs/execution-environments/r-support.rst:110]: It actually installs it automatically, or maybe just R studio does -> I'll move it to other options just to make sure.

File: docs/execution-environments/r-support.rst (hunk 2 above)
Comment (jakubczakon) [reply docs/execution-environments/r-support.rst:67]: ok, cool thanks

**Review by hubertjaworskineptune — APPROVED** [review]

**Pull request merged**